hooks/useNotification.tsx

```diff
@@ -17,6 +19,13 @@ const useNotification = () => {
         setNotifications((notificationsPrev) => notificationsPrev.filter((n) => n.id !== id))
     }
 
-    return { notifications, addNotification, removeNotification }
+    const populateNotificationError = (action: ActionNotification, newError: string) => {
```

Comment:
I would rename this a bit 
```suggestion
    const addCandyMachineNotificationError = (action: ActionNotification, error: string) => {
```